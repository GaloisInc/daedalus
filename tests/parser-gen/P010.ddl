{- Tests GetByte
-}

def Main = { x = UInt8, y = UInt8 }
