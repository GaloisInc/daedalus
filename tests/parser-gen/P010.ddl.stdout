module P010
 
--- Imports:
 
--- Type defs:
type P010.Main = { x: uint 8
                 ; y: uint 8
                 }
 
--- Rules:
 
P010.Main : Grammar P010.Main =
  do (x : uint 8) <- Match UInt8
     (y : uint 8) <- Match UInt8
     pure {x = x,
           y = y}
 
P010.Main_3 : Grammar {} =
  do @Match UInt8
     @Match UInt8
